pagopa/io-app
# chore(IT Wallet): [SIW-1284] Add IT Wallet discovery banner for trial users

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5934

## Short description
This PR adds the display of the IT Wallet discovery banner for trial users

## List of changes proposed in this pull request
- Added `isTrialActiveSelector` selector, which allows to know if the user is enabled for a trial
- Added `ItwDiscoveryBanner` component
- Added banner to messages' home
- Added banner to wallet's home
- Removed unused component `features/itwallet/common/components/FooterStackButton`

## How to test
Static checks should pass.
[Using the `io-dev-server`](https://github.com/pagopa/io-dev-api-server/pull/396), go to **Profile > Playgrounds > IT Wallet**. Make sure you have the IT Wallet trial enabled.
You should be able to display the banner in the Messages and Wallet screens. The banner can be closed but will appear again the next time you launch the app.

## Preview

https://github.com/pagopa/io-app/assets/6160324/e4df73fe-3c03-4dac-b52d-a705c942dc2b
